src/util/rocksdb/mod.rs

```diff
@@ -281,11 +286,13 @@ pub fn get_engine_compression_ratio_at_level(
     None
 }
 
+/// Get the number of files at given level of given cf.
```

Comment:
Can you expand "cf" to something more meaningful? What's a "cf"? It probably stands for something

Reply:
It stands for `column family` here.